Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/TargetsTests/RazorTargetTests.cs

```diff
@@ -96,7 +115,8 @@ public void Razor_ExcludedProject_NoErrorLog()
   <SonarQubeTempPath>{rootInputFolder}</SonarQubeTempPath>
   <SonarQubeExclude>true</SonarQubeExclude>
   <SonarErrorLog>OriginalValueFromFirstBuild.json</SonarErrorLog>
-</PropertyGroup>";
+</PropertyGroup>
```

Comment:
`RazorCompileOnBuild` was here from previous commits and wasn't replaced with `RazorComile`. It should be present to make sure this is not executed for correct reasons.